HOWTO_android.md

```diff
@@ -0,0 +1,637 @@
+```
+
+When this option is enabled, an allocation with a size equal to that of the
+object being mapped is made once on the first map and is not freed until the
+object is destroyed.
+This option is intended to be used with applications that frequently map and
+unmap large memory ranges, to avoid frequent allocation and copy operations that
+can have a negative impact on performance.
+
+Next, we will force page guard buffers are aligned to pages:
+
+```bash
+adb shell "debug.gfxrecon.page_guard_align_buffer_sizes true"
+```
+
+This option overrides the Vulkan API calls that report buffer memory properties
+to report that buffer sizes and alignments must be a multiple of the system page
+size.
+This option is intended to be used with applications that perform CPU writes and
+GPU writes/copies to different buffers that are bound to the same page of mapped
+memory, which may result in data being lost when copying pages from the
+`page_guard` shadow allocation to the real allocation.
+This data loss can result in visible corruption during capture.
+Forcing buffer sizes and alignments to a multiple of the system page size
+prevents multiple buffers from being bound to the same page, avoiding data loss
+from simultaneous CPU writes to the shadow allocation and GPU writes to the real
+allocation for different buffers bound to the same page.
+
+After making these changes, re-open the application and re-attempt recapture.
+
+
+### 7. Verify the Capture File
+
+I noticed in my capture `adb logcat` output that the file was recorded to
+`/sdcard'.
+**NOTE**: It actually lists the full file in the message like the following:
+
+```text
+gfxrecon: Recording graphics API capture to /sdcard/gfxrecon_capture_frames_500_through_700_20221211T130328.gfxr
+```
+
+But, let's say you didn't know, so double check that folder:
+
+```bash
+adb shell ls /sdcard
+```
+
+The output might look like this:
+
+```text
+Alarms
+Android
+Audiobooks
+DCIM
+Documents
+Download
+Movies
+Music
+Notifications
+Pictures
+Podcasts
+Recordings
+Ringtones
+boot.img
+com.khronos.vulkan_samples
+gfxrecon_capture_frames_500_through_700_20221211T130328.gfxr
+libVkLayer_gfxreconstruct.so
+libVkLayer_khronos_validation.so
+libc++_shared.so
+```
+
+There's our file right there.
+
+### 8. Disable The GfxReconstruct layer
+
+We want to make sure that the layer is not present for any further work we
+are going to do.
+In this case, we're going to check our replay, so disable any active debug
+layers so they don't interfere with the tool.
+
+```bash
+adb shell settings put global enable_gpu_debug_layers 0
+adb shell settings put global gpu_debug_layers ""
+```
+
+### 9. Install the replay application
+
+Now, we need to install the Replay application that we built as part of the
+GfxReconstruct source.
+
+First, let's make sure we don't have an old version lying around:
+
+```bash
+adb uninstall replay-debug.apk
+```
+
+Then, from the top of the source pulled down from the repo by using the
```

Comment:
How about "Then, install the replay APK from the root of the built source tree by using the `gfxrecon.py` script:"